src/modules/invite/dto/create-invite.dto.ts

```diff
@@ -0,0 +1 @@
+export class CreateInviteDto {}
```

Comment:
Why is this empty?

Reply:
I have now removed this file and implemented invite dto